### PR TITLE
feat: added --help command

### PR DESCRIPTION
**Pull request by abhishekpaturkar**
Capture

### CODE REVIEW

File: src/cli/index.ts [hunk 1]
```diff
@@ -4,6 +4,10 @@ import * as yargs from 'yargs';
 
 // Parse command-line arguments using yargs
 const argv = yargs
+   .usage('Usage: $0 [options]')
```
Comment (AmalChandru) [inline src/cli/index.ts:7]: Can you try to indent these to align with all other options?

File: src/cli/index.ts [hunk 2]
```diff
@@ -15,6 +19,10 @@ const argv = yargs
   .option('output', { type: 'string', default: 'plain', choices: ['plain', 'json'], describe: 'Output format' })
   .argv as yargs.Arguments;
 
+if (!argv._.length || argv._.includes('help')) {
+  yargs.showHelp();
+  process.exit(0);
+}
```
Comment (AmalChandru) [inline src/cli/index.ts:25]: Can you add a new line after this?

File: src/cli/index.ts [hunk 3]
```diff
@@ -27,6 +35,7 @@ const options: BenchmarkOptions = {
   output: argv.output as 'plain' | 'json',
 };
 
+
```
Comment (AmalChandru) [inline src/cli/index.ts:38]: Also please remove the extra new line here.

File: src/cli/index.ts
Comment (AmalChandru) [inline src/cli/index.ts:1]: The changes looks good. Suggested some minor formatting changes. Also can you update `README.md` to occupy this `--help` option?

**Review by AmalChandru — APPROVED** [review]

**Comment by abhishekpaturkar** [comment]
Done with changes and also updated README.md

**Comment by AmalChandru** [comment]
Thanks for the PR

**Comment by AmalChandru** [comment]
Hey, there was a small catch in your `--help` implementation. 
The original condition `argv._.length === 0` was causing the help to be shown even when all required options were provided. This is because yargs doesn't use positional arguments in this case, so` argv._`  was always empty. 
 [fix](https://github.com/AmalChandru/metrics-monkey/commit/3e36590c5dafba03b4394fa87f4e174d90305300)

**Comment by abhishekpaturkar** [comment]
Can you share me the ss to recreate the issue

**Comment by AmalChandru** [comment]
I have fixed that bro. You can test it using the older version in which you implemented `--help` option

**Pull request merged**